cmd/network/bridge/daemonset.yml

```diff
@@ -0,0 +1,37 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  labels:
+    name: device-plugin-network-bridge
+  name: device-plugin-network-bridge
+spec:
+  selector:
+    matchLabels:
+      name: device-plugin-network-bridge
+  template:
+    metadata:
+      labels:
+        name: device-plugin-network-bridge
+    spec:
+      hostNetwork: true
+      hostPID: true
+      containers:
+      - name: device-plugin-network-bridge
+        image: phoracek/device-plugin-network-bridge:latest
+        securityContext:
+          privileged: true
+        env:
+        - name: BRIDGES
+          value: "br0"
+        volumeMounts:
+          - name: var-run
+            mountPath: /var/run
```

Comment:
Why is this needed from the host?

Reply:
For docker.sock. I use docker to find out which container has just been crated.